### PR TITLE
Added Maker Made 300x Printer

### PR DESCRIPTION
**Pull request by skilescm**

### CODE REVIEW

File: resources/extruders/maker_made_300x_extruder_0.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (konskarm) [inline resources/extruders/maker_made_300x_extruder_0.json:1]: The extruder file should end in ".def.json".

File: resources/definitions/maker_made_300x.def.json [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        "visible": true,
+        "author": "DragonJe",
+        "manufacturer": "Maker Made",
+        "file_formats": "text/x-gcode",
+        "platform_offset": [0, 0, 0],
+        "has_materials": true,
+        "has_variants": false,
+        "has_machine_materials": true,
+        "preferred_quality_type": "normal",
+        "has_machine_quality": true,
```
Comment (konskarm) [inline resources/definitions/maker_made_300x.def.json:15]: As stated in https://github.com/Ultimaker/Cura/wiki/Definition-Files-Explained, the "has_machine_quality" should be set to true only if you have defined machine-specific quality files in the `resourses/quality/<machine_name>` folder.

**Review by konskarm — CHANGES_REQUESTED** [review]

File: resources/definitions/maker_made_300x.def.json [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "DragonJe",
+        "manufacturer": "Maker Made",
+        "file_formats": "text/x-gcode",
+        "platform_offset": [0, 0, 0],
+        "has_materials": true,
+        "has_variants": false,
+        "has_machine_materials": true,
```
Comment (konskarm) [inline resources/definitions/maker_made_300x.def.json:13]:
```suggestion
```
No longer used.

File: resources/definitions/maker_made_300x.def.json [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+    "overrides": {
+        "machine_name": {"default_value": "Maker Made 300x"},
+        "machine_width": {"default_value": 300},
+        "machine_height": {"default_value": 400},
+        "machine_depth": {"default_value": 300},
+        "machine_head_with_fans_polygon": {"default_value": [[-30, 34],[-30, -32],[30, -32],[30, 34]]},
+        "gantry_height": {"value": "30"},
+        "machine_heated_bed": {"default_value": true},
+        "material_diameter": {"default_value": 1.75},
```
[the remark this answers is not in the document]
Comment (skilescm) [reply resources/definitions/maker_made_300x.def.json:30]: Deleted line 77

**Review by skilescm — COMMENTED** [review]
All requested changes have been made.

**Comment by skilescm** [comment]
All requests fulfilled.

**Comment by konskarm** [comment]
All issues have been fixed. The printer is added in Cura and slices without a problem. 
The Maker Made 300x definition will become available to be used in the next Cura release.

**Pull request merged**